pkg/metrics/metrics.go

```diff
@@ -165,6 +165,12 @@ var (
 		Name:      "udf_write_total",
 		Help:      "Total number of Messages Written by UDF",
 	}, []string{LabelVertex, LabelPipeline, LabelVertexType, LabelVertexReplicaIndex, LabelPartitionName})
+
+	DroppedMessageCounter = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Do we already have `DropMessagesCount` in the same file?

Reply:
That one is mainly for dropping on buffer full...

use `ud_drop_total`?

Reply:
or `ud_dropped_messages_total`